app/controllers/employees/rewards/orders_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Employees
+  module Rewards
+    class OrdersController < EmployeesController
+      before_action :set_reward
+      def new
+        @order = Order.new
+      end
+
+      def create
+        # flash[:alert]= 'You do not have enough points for that reward'
```

Comment:
You need to review the code on your own, before sending it for review to someone else.
Why you left this comment here?

Reply:
Sorry about that. Already corrected.